src/marqo/core/unstructured_vespa_index/unstructured_vespa_index.py

```diff
@@ -318,9 +323,14 @@ def _to_vespa_hybrid_query(self, marqo_query: MarqoHybridQuery) -> Dict[str, Any
         summary = unstructured_common.SUMMARY_ALL_VECTOR if marqo_query.expose_facets \
             else unstructured_common.SUMMARY_ALL_NON_VECTOR
 
+        if self._marqo_index_version >= self._HYBRID_SEARCH_MINIMUM_VERSION:
```

Comment:
We don't support hybrid for these versions to why this branching?

Reply:
fixed